projects/imgix-ng/src/lib/imgix.component.ts

```diff
@@ -68,35 +64,28 @@ export class ImgixComponent implements AfterViewChecked {
   }
   private _disableVariableQuality: boolean = false;
 
-  @Input() htmlAttributes?: Object;
-
-  constructor(@Inject(ImgixConfigService) private config: ImgixConfig) {
-    this.client = this.createImgixClient(this.config);
-  }
-
-  private createImgixClient(config: ImgixConfig): ImgixClient {
-    const client = new ImgixClient({
-      domain: config.domain,
-      includeLibraryParam: false,
-    });
-
-    if (!(config.includeLibraryParam === false)) {
-      (client as any).settings.libraryParam = `ng-${VERSION}`;
-    }
+  @Input() htmlAttributes?: Record<string, string>;
 
-    return client;
+  constructor(
+    @Inject(ImgixConfigService) private config: ImgixConfig,
+    private elementRef: ElementRef<HTMLImageElement>,
```

Comment:
Does this get used?

Reply:
Yep, in `setOtherAttributes()` and `setSrcAndSrcsetAttributes()`